package.json

```diff
@@ -6,23 +6,21 @@
     "ng": "ng",
     "start": "ng serve",
     "build": "ng build --prod",
-    "sw-build": "ng build --prod && node build/fix-sw",
     "test": "ng test",
     "lint": "ng lint",
     "e2e": "ng e2e"
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^7.0.0",
-    "@angular/common": "^7.0.0",
-    "@angular/compiler": "^7.0.0",
-    "@angular/core": "^7.0.0",
-    "@angular/forms": "^7.0.0",
-    "@angular/http": "^7.0.0",
-    "@angular/platform-browser": "^7.0.0",
-    "@angular/platform-browser-dynamic": "^7.0.0",
-    "@angular/router": "^7.0.0",
-    "@angular/service-worker": "^7.0.0",
+    "@angular/animations": "^7.0.2",
+    "@angular/common": "^7.0.2",
+    "@angular/compiler": "^7.0.2",
+    "@angular/core": "^7.0.2",
+    "@angular/forms": "^7.0.2",
+    "@angular/http": "^7.0.2",
+    "@angular/platform-browser": "^7.0.2",
+    "@angular/platform-browser-dynamic": "^7.0.2",
+    "@angular/router": "^7.0.2",
```

Comment:
Still need service worker, make sure implementation works with `yarn`

Reply:
having trouble with yarn:

`info There appears to be trouble with your network connection. Retrying...`

I think ryan had this problem too. i think its windows specific. will try on my linux distro

Reply:
finished